### PR TITLE
Add recentclicks2 and recent_srs to cookies

### PR DESCRIPTION
**Pull request by jamesmaa**
👓 : @dwick

### CODE REVIEW

File: src/routes.jsx [hunk 1]
```diff
@@ -461,8 +474,13 @@ function routes(app) {
       commentId: ctx.params.commentId,
     });
 
-    const visited = getVisitedPosts();
-    setVisitedPosts([ctx.params.listingId].concat(visited));
+    const data = props.data;
+    const user = data.get('loggedOutUser') || data.get('user');
```
Comment (dwick) [inline src/routes.jsx]: if the user is logged out what happens here? don't we want it to be `_cookie_name` in that case?

File: src/lib/visitedSubreddits.es6.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+// Return an array of ids of the subreddits the user has visited recently.
+// Return [] if we are unable to access such a list.
+export function getVisitedSubreddits(username) {
+  if (!localStorageAvailable()) {
+    return [];
+  }
+
+  const key = [username, constants.RECENT_SUBREDDITS_KEY].join('_');
+  const visitedString = global.localStorage.getItem(key);
```
Comment (dwick) [inline src/lib/visitedSubreddits.es6.js]: we want to be able to grab these from the cookies as well if they were set by r2

**Comment by dwick** [comment]
💅

File: src/routes.jsx (hunk 1 above)
Comment (jamesmaa) [inline src/routes.jsx]: The assumption I'm working with here is that when a user is loggedout, data will have `loggedOutUser`, whereas when a user is loggedIn, data will have `user`.

loggedOut: 
image

loggedIn:
image

Comment (jamesmaa) [inline src/routes.jsx]: the cookie name is set in `setVisitedSubreddits`. it'll save as `_recent_srs`

**Comment by jamesmaa** [comment]
💇

File: src/routes.jsx [hunk 3]
```diff
@@ -479,7 +479,7 @@ function routes(app) {
     user.then(user => {
       const username = user.body.name;
       const visited = getVisitedPosts(username);
-      setVisitedPosts(username, [ctx.params.listingId].concat(visited));
+      setVisitedPosts(username, ['t3_' + ctx.params.listingId].concat(visited));
```
Comment (dwick) [inline src/routes.jsx]: 🔕 are there no constants for types so you dont have to remember them?

**Comment by dwick** [comment]
🐟 lgtm

**Comment by jamesmaa** [comment]
deploy broke :( fixing

**Pull request closed without merging**